.storybook/preview.tsx

```diff
@@ -44,9 +30,7 @@ const AppDecorator: Exclude<Preview["decorators"], undefined>[0] = (
 
   return (
     <L10nProvider bundleSources={l10nBundles}>
-      <ReactAriaI18nProvider locale={getLocale(l10nBundles)}>
-        {storyFn()}
-      </ReactAriaI18nProvider>
+      <ReactAriaI18nProvider locale={"en"}>{storyFn()}</ReactAriaI18nProvider>
```

Comment:
Not sure it needs the curly braces in `locale={"en"}`:
```suggestion
      <ReactAriaI18nProvider locale="en">{storyFn()}</ReactAriaI18nProvider>
```

Reply:
Fairly sure it does not: 95f9e8acf.